### DISCUSSION TITLE
Vercel Provider

### DISCUSSION THREAD

**Discussion by NCCoder1** [xtekky/gpt4free#938] [Q&A]
How do I fix this? I have the right config and all for Vercel, but when I run it all I get is <generator object Vercel.create_completion at 0x7f5c2a4f82e0>

**Comment by hlohaus** [comment]
You have a generator object. You can iterate it to get the values:
```
for chunk in response:
    print(chunk, end='', flush=True)
```
```
return ''.join([chunk for chunk in response])
```

**Reply by NCCoder1** [reply]
Thanks!

**Reply by NCCoder1** [reply]
Wait, could u tell me how to implement it in: 

import g4f
from g4f.Provider import (
    Vercel
)

print(g4f.Provider.Ails.params)  

response = g4f.ChatCompletion.create(
    model="gpt-3.5-turbo",
    provider=g4f.Provider.Vercel,
    messages=[{"role": "user", "content": "Hello world"}],
    stream=True,
)
